ONSdigital/response-operations-ui
# Us211 display respondent for seft rus ce in response status

# Motivation and Context
In response-operation-ui, we want to be able to see the respondent name on the responses status page when a external user has completed a collection exercise. For this to happen, response status needs to get a list of case events that are completed for that collex and get the respondent that completed it.

# What has changed
- Added respondent to the response status page
- Added tests

# How to test?
- Run `make test` and all tests should pass
- Complete a SEFT collection exercise. When completed, look on response ops response status page and the completed collex should have a respondent name attached to it.
- With a completed eQ collex, when viewed it should show a blank respondent and not bring up a 500 error page.
- Run with acceptance tests - [Acceptance tests](https://github.com/ONSdigital/rasrm-acceptance-tests/pull/222)

# Links
[Trello](https://trello.com/c/1rcfmtok)
